docs/tutorial/stackup/2. smart-contract/2.2 Write-Voting-Smart-Contract.md

```diff
@@ -0,0 +1,135 @@
+---
+sidebar_position: 2
+---
+
+# Write Voting Smart Contract
+
+In this step, we will first be implementing the RPC methods and message definitions of our Voting dApp in the **BuildersDAO.proto** file!
+
+Proceed to open the **src/Protobuf/contract/BuildersDAO.proto** and overwrite the existing file contents with the following code snippet.
+
+```protobuf showLineNumbers
+syntax = "proto3";
+
+import "aelf/core.proto";
+import "aelf/options.proto";
+import "google/protobuf/empty.proto";
+import "acs12.proto";
```

Comment:
If user is not working in codespace, this needs to import from the local directory instead.